api/services/user_test.go

```diff
@@ -127,10 +127,14 @@ func TestUpdateDataUser(t *testing.T) {
 		},
 	}
 
-	for _, tc := range tests {
-		tc.requiredMocks()
-		returnedFields, err := s.UpdateDataUser(ctx, tc.user, tc.updateUser.ID)
-		assert.Equal(t, tc.expected, Expected{returnedFields, err})
+	for _, test := range tests {
+		tc := test
```

Comment:
Same here; avoid the alias.

Reply:
This is required by linter

https://github.com/shellhub-io/shellhub/actions/runs/2059037104